cloud-controller-manager/do/kubernetes_lb_validator.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2023 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package do
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/digitalocean/godo"
+	"github.com/go-logr/logr"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	lbIDAnnotation string = "kubernetes.digitalocean.com/load-balancer-id"
```

Comment:
Please don't copy but export and reuse [this one](https://github.com/digitalocean/digitalocean-cloud-controller-manager/blob/0fea136d928d94a435a3d89ecb03b212c9ee4e3b/cloud-controller-manager/do/lb_annotations.go#L22).